env0/utils.go

```diff
@@ -50,15 +50,25 @@ func stringInSlice(str string, strs []string) bool {
 	return false
 }
 
-func reasourceDataGetValue(fieldName string, d *schema.ResourceData) interface{} {
+func reasourceDataGetValue(fieldName string, omitEmpty bool, d *schema.ResourceData) interface{} {
 	dval := d.Get(fieldName)
 	if dval == nil {
 		return nil
 	}
 
-	_, okInt := dval.(int)
+	ival, okInt := dval.(int)
+
+	if omitEmpty && okInt && ival == 0 {
```

Comment:
handles omit empty for empty string and or zero int.
Up until now, it wasn't required. The first use-case it's needed, so added it.